model/Core/Vocabularies/RelationshipType.md

```diff
@@ -12,73 +12,79 @@ Provides information about the relationship between two Elements.
-- expandedFromArchive: Every `to` Element is an artifact expanded from the `from` archive file
+- hasConcludedLicense: The `from` software Artifact is concluded by the SPDX data creator to be governed by each `to` license
+- contains: The `from` Element contains each `to` Element
+- configOf: The `from` Element is a configuration applied to each `to` Build during a LifecycleScopeType period
+- copiedTo: The `from` Element has been copied to each `to` Element
+- hasDataFile: The `from` Element treats each `to` Element as a data file
+- hasDeclaredLicense: The `from` software Artifact was discovered to actually contain each `to` license, for example as detected by use of automated tooling.
+- hasDependencyManifest: The `from` Element has manifest files that contain dependency information in each `to` Element
+- dependsOn: The `from` Element depends on each `to` Element
+- descendantOf: The `from` Element is a descendant of each `to` Element
+- describes: The `from` Element describes each `to` Element. This can be used to denote the root(s) of a tree of elements contained in an SBOM.
+- hasDevDependency: The `from` Element has a development dependency on each `to` Element
+- hasDevTool: The `from` Element uses each `to` Element as a development tool
+- hasDistributionArtifact: The `from` Element is distributed as an artifact in each Element `to`, (e.g. an RPM or archive file)
+- hasDocumentation: The `from` Element is documented by each `to` Element
+- doesNotAffect: (Security/VEX) The `from` Vulnerability has no impact on each `to` Element
+- hasDynamicLink: The `from` Element dynamically links in each `to` Element
+- hasExample: Every `to` Element is an example for the `from` Element (`from` hasExample `to`)
+- hasEvidence: (Dataset) Every `to` Element is considered as evidence for the `from` Element (`from` hasEvidence `to`)
+- expandsTo: The `from` archive expands out as an artifact described by each `to` Element
 - exploitCreatedBy: (Security) Designates an agent has created an exploit against a vulnerability
-- fileAdded: Every `to` Element is is a file added to the `from` Element
-- fileDeleted: Every `to` Element is a file deleted from the `from` Element
-- fileModified: Every `to` Element is a modification of the `from` Element
+- hasAddedFile: Every `to` Element is is a file added to the `from` Element (`from` hasAddedFile `to`)
+- hasDeletedFile: Every `to` Element is a file deleted from the `from` Element (`from` hasDeletedFile `to`)
+- modifiedBy: The `from` Element is modified by each `to` Element
 - fixedBy: (Security) Designates a vulnerability has been fixed by an agent
 - fixedIn: (Security/VEX) A vulnerability has been fixed in one or more elements
 - foundBy: (Security) Designates an agent was the original discoverer of a security vulnerability
-- generates: Every `to` Element is generated from the `from` Element
+- generates: The `from` Element generates each `to` Element
```

Comment:
When I look at the 2.2 relationships in use:
Relationship: SPDXRef-File-libkernel.a GENERATED_FROM DocumentRef-zephyr:SPDXRef-File-main-weak.c
Relationship: SPDXRef-File-libkernel.a GENERATED_FROM DocumentRef-zephyr:SPDXRef-File-banner.c
Relationship: SPDXRef-File-libkernel.a GENERATED_FROM DocumentRef-zephyr:SPDXRef-File-busy-wait.c

so I think this should have the properties swapped to be consistent.

generatedBy may be clearer and consistent with other relationships? 
so something like

generatedBy: The `from` Element is generated by each of the `to` Element(s)

Reply:
Right. I was trying to preserve the originally stated order, but I'm happy to flip it around if we like it better. `generatedFrom` is fine, as it is still clear what the directionality should be from the name.

Reply:
From discussion,  ok to flip order and match SPDX 2.

Reply:
Will be flipped in a separate PR